ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -365,7 +368,12 @@ bool ManageTokensController::lessThan(const QString& lhsSymbol, const QString& r
     leftPos = leftVisible ? leftPos : INT_MAX;
     rightPos = rightVisible ? rightPos : INT_MAX;
 
-    return leftPos <= rightPos;
+    if (leftPos < rightPos)
+        return -1;
+    else if (leftPos > rightPos)
+        return 1;
+    else
+        return 0;
```

Comment:
```suggestion
    if (leftPos < rightPos)
        return -1;
    if (leftPos > rightPos)
        return 1;
    return 0;
```
bit shorter alternative